py_trees/tests.py

```diff
@@ -34,15 +34,16 @@ def pre_tick_visitor(behaviour_tree):
 def tick_tree(tree,
               from_tick,
               to_tick,
-              visitor=None,
+              *,
```

Comment:
BTW (non-blocking) looks like all tests pass without '*' this since you renamed everything, I think it's okay to remove the *.

Reply:
Oh, didn't do it for that. Left it there to force me to use keywords when the positional trail becomes excessive (and non-obvious).